metroplex.js

```diff
@@ -92,32 +92,16 @@ Metroplex.readable('parse', function parse(server) {
 Metroplex.readable('register', function register(address, fn) {
   var metroplex = this;
 
-  metroplex.address = this.parse(address || metroplex.address);
+  metroplex.address = metroplex.parse(address || metroplex.address);
   if (!metroplex.address) {
     if (fn) fn();
     return this;
   }
 
-  metroplex.leverage.annihilate(metroplex.address, function annihilate(err) {
-    if (err) {
-      if (fn) return fn(err);
-      return metroplex.emit('error', err);
-    }
-
-    metroplex.redis.multi()
-      .setex(metroplex.namespace + metroplex.address, metroplex.interval, Date.now())
-      .sadd(metroplex.namespace +'servers', metroplex.address)
-    .exec(function register(err) {
-      if (err) {
-        if (fn) return fn(err);
-        return metroplex.emit('error', err);
-      }
-
-      metroplex.emit('register', metroplex.address);
-      metroplex.setInterval();
-
-      if (fn) fn(err, metroplex.address);
-    });
+  metroplex.redis.psetex(metroplex.namespace + 'server:' + metroplex.address, metroplex.interval, Date.now(), function(err, result) {
```

Comment:
I know that redis commands can fail only if called with a wrong syntax or against keys holding the wrong data type, but I would handle the error anyway, in the same way it was handled before.